cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MultiMeasureServiceTest.java

```diff
@@ -931,4 +931,22 @@ void MultiMeasure_EightMeasures_ReporterNotAcceptedResource() {
                 .hasSubjectReference("Practitioner/tester")
                 .hasReporter("Patient/male-2022"));
     }
+
+    @Test
+    void MultiMeasure_EightMeasures_SubjectOrganization() {
+        var when = GIVEN_REPO
+                .when()
```

Comment:
Question: Is this test notEnabling partOf?
Suggestion: add comment on what functionality is being validated